B2C-Tfp-Issuer.md

```diff
@@ -0,0 +1,14 @@
+# Azure AD B2C issuer claim
```

Comment:
Better title?

Reply:
i agree, i would put this in the wiki.

Reply:
Yes, it will be in the wiki. Added here just to get comments.